doc/nrf/device_guides/nrf53/multi_image_nrf5340.rst

```diff
@@ -1,12 +1,15 @@
 .. _ug_nrf5340_multi_image:
 
-Multi-image builds on the nRF5340 DK
-####################################
+Multi-image builds on the nRF5340 DK (child/parent) (deprecated)
+################################################################
 
 .. contents::
    :local:
    :depth: 2
 
+.. important::
+    |sysbuild_related_deprecation_note|
```

Comment:
This note should be edited to point toward NCS docs discussing sysbuild, as the reader might miss them if starting its sysbuild journey from Zephyr docs.

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/config_and_build_system.html#sysbuild